frontend-react/src/components/MessageTracker/MessageReceivers.tsx

```diff
@@ -393,67 +394,71 @@ export const MessageReceivers = ({ receiverDetails }: MessageReceiverProps) => {
                         <tr>
                             <MessageReceiversCol
                                 {...commonProps}
-                                columnHeaderTitle={ColumnDataEnum.Name}
+                                columnHeader={"name"}
```

Comment:
Might be good to call this prop `columnKey` or something for clarity since it'll be the identifier for any column, whether we're in the context of the headers or an individual data cell.

Reply:
should we override the default key property to just be required and use that?

Reply:
followup: avoiding the key prop, will use columnKey

Reply:
For context, my feeling was that we shouldn't overload the `key` prop since that's specifically for React rendering stability/optimizations.  Anything more than that would be outside the purview of this prop (i.e., plucking cell data) so we decided to keep it more explicit with `columnKey`.